mesheryctl/internal/cli/root/app/offboard.go

```diff
@@ -0,0 +1,60 @@
+package app
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var offboardCmd = &cobra.Command{
+	Use:   "offboard",
+	Short: "Offboard application",
+	Long:  `Offboard application will trigger deletion of the application file`,
+	Args:  cobra.MinimumNArgs(0),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
+		if err != nil {
+			return errors.Wrap(err, "error processing config")
+		}
+
+		// Read file
+		fileReader, err := os.Open(file)
+		if err != nil {
+			return errors.New(utils.SystemError(fmt.Sprintf("failed to read file %s", file)))
+		}
+
+		client := &http.Client{}
+		req, err := http.NewRequest("DELETE", mctlCfg.GetBaseMesheryURL()+"/api/experimental/application/deploy", fileReader)
+		if err != nil {
+			return err
+		}
+
```

Comment:
we missed grabbing `tokenPath` like listCmd.

Reply:
yeah, same!

Reply:
ok